lib/Target/AVR/AVRBranchSelector.cpp

```diff
@@ -180,6 +180,9 @@ bool AVRBSel::runOnMachineFunction(MachineFunction &Fn)
           }
         }
 
+        if(isCondBranch(Opc))
+          BranchSize -= 2; // take the size of the current instruction.
+
```

Comment:
I'm not really sure how this made it into the PR, I think a bad rebase. I also had to remove another commit.